.github/workflows/release.yaml

```diff
@@ -0,0 +1,94 @@
+name: Ion Hash DotNet Release
+
+on:
+  release:
+    types: [ created ]
+
+jobs:
+  test:
+    name: Test
+    if: startsWith(github.ref, 'refs/tags/')
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ windows-latest ]
+        dotnet: ['2.1']
```

Comment:
We have a known issue that Ion Hash DotNet is not 100% compatible with core Net 3.1 now so we run everything in a 2.1 environment. See https://github.com/amzn/ion-hash-dotnet/pull/49.